packages/salesforcedx-vscode-soql/src/commands/soqlBuilderToggle.ts

```diff
@@ -20,9 +20,5 @@ export async function soqlBuilderToggle(doc: vscode.Uri): Promise<void> {
     ? BUILDER_VIEW_TYPE
     : EDITOR_VIEW_TYPE;
 
-  vscode.commands.executeCommand(
-    OPEN_WITH_COMMAND,
-    vscode.Uri.file(doc.fsPath),
```

Comment:
this just produces a Uri, which is what `doc` already is.  So I just use that as is since the [openWith command takes a Uri](https://github.com/microsoft/vscode/blob/afca44535df55707d5f0b1dd65098060b448c8a8/src/vs/workbench/contrib/customEditor/browser/customEditors.ts#L137).